src/components/Dropdown/Dropdown.tsx

```diff
@@ -70,7 +69,7 @@ export const Dropdown: FC<DropdownProps> = React.memo(
       });
 
       const [closing, setClosing] = useState<boolean>(false);
-      const [dropdownId] = useState<string>(uniqueId('dropdown-'));
+      const dropdownId: string = uniqueId('dropdown-');
```

Comment:
i thought useState was supposed to maintain value after getting initialized unless the setter gets called. Was this not to my understanding?